### DISCUSSION TITLE
求泰拉瑞亚

### DISCUSSION THREAD

**Discussion by CoderHentai** [Gamuxorg/bbs#364] [投递和请求游戏]
在海盗湾找到了一份资源 https://thepiratebay.org/torrent/20561506/Terraria_1.3.5.3_License_GOG_(LINUX) 但下载不了

**Comment by hyh19962008** [comment]
站里有泰拉瑞亚的https://www.linuxgame.cn/terraria

**Comment by CoderHentai** [comment]
谢谢，我眼瞎了没看到
